dev-cpp/tclap/tclap-1.2.2.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
7

Reply:
Hi Mikle,
I switched to the latest EAPI and unless I overlooked something in the documentation, no other changes are needed to support EAPI 7. I rechecked and the updated ebuild installs fine.
Thanks for your feedback.